### PR TITLE
Add prompt for the user to give their ETH address at the end of the round

### PR DESCRIPTION
**Pull request by julesdesmit**

### CODE REVIEW

**Comment by ibaryshnikov** [comment]
What do you think about adding it as an optional cli argument instead?

**Comment by julesdesmit** [comment]
@ibaryshnikov I think it may be good to at least have the user pause and read the message/know that the option is there, rather than it being a CLI argument and possibly as a sidenote in a step-by-step guide or something. Maybe it makes sense to move the prompt to the start of the round?

**Comment by ibaryshnikov** [comment]
> I think it may be good to at least have the user pause and read the message/know that the option is there, rather than it being a CLI argument and possibly as a sidenote in a step-by-step guide or something

good point!

> Maybe it makes sense to move the prompt to the start of the round?

Sounds like a viable option

File: setup1-contributor/src/commands/contribute.rs [hunk 1]
```diff
@@ -750,21 +759,69 @@ impl Contribute {
             .error_for_status()?;
         Ok(())
     }
+
+    async fn prompt_eth_address<R: Rng + CryptoRng>(&self, auth_rng: &mut R) -> Result<()> {
+        println!(
+            "As a token of our appreciation, we would like to send an NFT to all participants, which shows you've participated in the Aleo setup ceremony. If you would like to receive such an NFT, please enter your ETH address! If not, just hit enter."
```
Comment (apruden2008) [inline setup1-contributor/src/commands/contribute.rs:765]: Thank you for participating in Aleo Setup. As a token of appreciation, we would like to send you a commemorative NFT. This NFT is procedurally generated, but represents your unique contribution. We hope it serves as a reminder of the important role that YOU played in bringing Aleo to life.

**Review by apruden2008 — COMMENTED** [review]
gtg pending the prompt update

**Review by apruden2008 — APPROVED** [review]

**Review by emmorais — APPROVED** [review]
LGTM

**Comment by julesdesmit** [comment]
Keeping this on draft while I fix the upload procedure

**Pull request merged**